source/vocab/platform.ttl

```diff
@@ -263,6 +264,47 @@
     owl:equivalentProperty void:inDataset;
     rdfs:subPropertyOf dc:isPartOf .
 
+# Technical  details and change descriptions {{{
+
+:technicalNote a owl:ObjectProperty;
+    rdfs:subPropertyOf :hasNote;
+    rdfs:domain :Record;
+    rdfs:range :TechnicalNote;
+    rdfs:label "technical note"@en, "systemteknisk anmärkning"@sv .
+
+:TechnicalNote a owl:Class;
+    rdfs:subClassOf :StructuredValue;
+    rdfs:label "Technical note"@en, "Systemteknisk anmärkning"@sv .
+
+:hasChangeNote a owl:ObjectProperty ;
+  rdfs:subPropertyOf :technicalNote ;
+  sdo:domainIncludes :Record ;
+  rdfs:range :ChangeNote ;
+  rdfs:label "Change note"@en, "Ändringsanmärkning"@sv .
+
+:ChangeNote a owl:Class ;
+  rdfs:subClassOf :TechnicalNote ;
+  rdfs:subClassOf [ a owl:Restriction ;
+      owl:onProperty :category ;
+      owl:allValuesFrom :ChangeCategory
+    ] ;
+  rdfs:label "Change note"@en, "Ändringsanmärkning"@sv .
```

Comment:
`:category :platform` ?
If we plan to have these inherited through `:integral` relations.

Reply:
Definitely `:platform`, and _probably_ either in `:integral`, or since they are in `:platform`, "hardcoded" usage.

Reply:
Addressed in b6fabb13125e2498e8f46f7c645cd728788b59e4